05/helpers/validator.js

```diff
@@ -0,0 +1,27 @@
+// import React from 'react';
+
+function validator(inputData, fieldsArr) {
+    // eslint-disable-next-line prefer-const
+    let errors = [];
+
+    // eslint-disable-next-line array-callback-return
+    fieldsArr.map((field) => {
+        const value = inputData[field.name];
+
+        if (field.required) {
+            if (value.length <= 2 && field.name !== 'email') {
```

Comment:
Ja bym nie wykluczał adresu email (2 znaki to też za mało dla niego :P)

Reply:
Jak jest błędny e-mail to pokazuje też email invalid, więc nie chciałam powtarzać dwóch błędów dla emaila ;)